model_api/cpp/models/src/classification_model.cpp

```diff
@@ -358,6 +359,27 @@ std::unique_ptr<ResultBase> ClassificationModel::get_hierarchical_predictions(In
     return retVal;
 }
 
+ov::Tensor ClassificationModel::reorder_saliency_maps(const ov::Tensor& source_maps) {
+    if (!hierarchical || source_maps.get_shape().size() == 1) {
```

Comment:
In which case source_maps.get_shape().size()==1 can be  true?

Reply:
If saliency map is an empty tensor, that's a sanity check